osu.Game/Screens/Play/SkipOverlay.cs

```diff
@@ -214,17 +216,21 @@ private class Button : OsuClickableContainer
             private Box background;
             private AspectContainer aspect;
 
+            private SampleChannel sampleConfirm;
+
             public Button()
             {
                 RelativeSizeAxes = Axes.Both;
             }
 
             [BackgroundDependencyLoader]
-            private void load(OsuColour colours)
+            private void load(OsuColour colours, AudioManager audio)
             {
                 colourNormal = colours.Yellow;
                 colourHover = colours.YellowDark;
 
+                sampleConfirm = audio.Sample.Get(@"SongSelect/confirm-selection");
```

Comment:
This should use `SkinnableSound` instead.

Reply:
Hmm... even though it's not within a skinnable context currently? Moving it will change depth so isn't feasible.

Reply:
Whaaaaat... Why is this not in a skinnable context already...

Reply:
Because only gameplay is skinnable for now. The skip button is not part of gameplay (it uses the menu cursor, for instance).